data/mods/Animatronics/speech.json

```diff
@@ -0,0 +1,183 @@
+[
+    {
+        "type" : "speech",
+        "speaker" : "mon_animatronic_fox_old",
+        "sound" : "\"Don't make me have ye walk the plank.\"",
+        "volume" : 40
+    },{
+        "type" : "speech",
+        "speaker" : "mon_animatronic_fox_old",
+        "sound" : "\"Welcome to the pirate's cove matey!\"",
+        "volume" : 40
+    },{
+        "type" : "speech",
+        "speaker" : "mon_animatronic_fox_old",
+        "sound" : "\"Arr, ahoy there maties! And welcom- welcom- welcome to tha pirate's cove.\"",
+        "volume" : 40
+    },{
+        "type" : "speech",
+        "speaker" : "mon_animatronic_fox_old",
+        "sound" : "\"It's me.\"",
+        "volume" : 40
+	},{
+        "type" : "speech",
+        "speaker" : "mon_animatronic_fox_old",
+        "sound" : "\"SAVE THEM.\"",
+        "volume" : 40
+	},{
+        "type" : "speech",
+        "speaker" : "mon_animatronic_fox_old",
+        "sound" : "\"HELP THEM\"",
+        "volume" : 40
+	},{
+        "type" : "speech",
+        "speaker" : "mon_animatronic_fox_old",
+        "sound" : "\"SKREEEEEEEEEEEEEE!\"",
+        "volume" : 75
+	},{
+        "type" : "speech",
+        "speaker" : "mon_animatronic_fox_old",
+        "sound" : "\"Dum de dum dum dum.\"",
+        "volume" : 30
+	},{
+        "type" : "speech",
+        "speaker" : "mon_animatronic_fox_old",
+        "sound" : "\"Ahoy there!\"",
+        "volume" : 40
+    },{
+        "type" : "speech",
+        "speaker" : "mon_animatronic_bunny_old",
+        "sound" : "\"SKREEEEEEEEEEEEEE!\"",
+        "volume" : 75
+    },{
+        "type" : "speech",
+        "speaker" : "mon_animatronic_bear",
+        "sound" : "\"They will know the joy of creation.\"",
+        "volume" : 40
+    },{
+        "type" : "speech",
+        "speaker" : "mon_animatronic_bear",
+        "sound" : "\"It's me.\"",
+        "volume" : 40
+    },{
+        "type" : "speech",
+        "speaker" : "mon_animatronic_bear",
+        "sound" : "\"SAVE THEM\"",
+        "volume" : 40
+	},{
+        "type" : "speech",
+        "speaker" : "mon_animatronic_bear",
+        "sound" : "\"HELP THEM\"",
```

Comment:
I read something about getting messages in the minigames along these lines.  *shrug*  Blob can trigger an existing voice-box, sure, but conscious speech not so much.